### PR TITLE
Add com.obsproject.Studio.Plugin.TransitionTable

### PR DESCRIPTION
**Pull request by exeldro**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.TransitionTable

File: com.obsproject.Studio.Plugin.TransitionTable.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.TransitionTable</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Transition Table Plugin</name>
+  <summary>Plugin for OBS Studio to add a Transition Table to the tools menu</summary>
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.TransitionTable.metainfo.xml:6]: Hm, how about something like `Fine-tune your transitions with a transition table`?

(I've been advocating for removing things like "Plugin for OBS Studio that ..." from summaries. Instead, I'm suggesting summaries that briefly explain what that plugin enables people to do.)

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
Sorry it took me so long to review this. Had one comment, it's fairly minor, but I wish we can work out a better summary before including this plugin.

**Pull request closed without merging**